lib/discourse_theme/cli.rb

```diff
@@ -12,14 +19,23 @@ def self.settings_file=(filename)
     end
 
     def usage
-      puts "Usage: discourse_theme COMMAND [--reset]"
-      puts
-      puts "discourse_theme new DIR - Creates a new theme in the designated directory"
-      puts "discourse_theme download DIR - Downloads a theme from the server and stores in the designated directory"
-      puts "discourse_theme upload DIR - Uploads the theme directory to Discourse"
-      puts "discourse_theme watch DIR - Watches the theme directory and synchronizes with Discourse"
-      puts
-      puts "Use --reset to change the configuration for a directory"
+      puts <<~USAGE
+      Usage: discourse_theme COMMAND [DIR] [OPTIONS]
+
+      Commands:
+        new DIR               - Creates a new theme in the specified directory.
+        download DIR          - Downloads a theme from the server and stores it in the specified directory.
+        upload DIR            - Uploads the theme from the specified directory to Discourse.
+        watch DIR             - Watches the theme in the specified directory and synchronizes any changes with Discourse.
+        rspec DIR [OPTIONS]   - Runs the RSpec tests in the specified directory.
```

Comment:
How easy would it be to provide an option to run against an existing local development environment? I think most CDCK devs would prefer that, and it would also unlock running theme system specs against discourse installations with certain plugins. 🤔

Reply:
Another reason I'm realising while trying this out: the discourse_test image is sloooooow when running on Apple Silicon with virtualisation, so being able to run against a regular dev environment would be especially useful.

(or we try and build an ARM version of the discourse_test image... but that's a whole project in itself)

Reply:
> How easy would it be to provide an option to run against an existing local development environment?

In theory super easy because all you really need to do is to just `bundle exec rspec` in the `discourse` repository and then just point it at the path of the theme's spec folder. I'll see if I integrate this into the `discourse_theme rspec` command. We might have to set some sort of ENV variable in order to tell the `discourse_theme` CLI where the discourse repository is located though.